### ISSUE TITLE
Dependency with ClassificationBundle should be removed

### ISSUE THREAD

**Issue by nicolasricci** [sonata-project/SonataMediaBundle#830]
There is currently a dependency with the SonataClassificationBundle  in the `sonata.media.admin.media` service

https://github.com/sonata-project/SonataMediaBundle/blob/master/Resources/config/doctrine_orm_admin.xml#L30

I guess this should be addressed differently.

**Comment by ruscon** [comment]
```
composer update "sonata-project/media-bundle"
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for sonata-project/media-bundle dev-master -> satisfiable by sonata-project/media-bundle[dev-master].
    - sonata-project/media-bundle dev-master requires sonata-project/classification-bundle ~2.3 -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.
```

**Comment by gabrielAnglada** [comment]
Please, is this fixed?

**Comment by JHGitty** [comment]
https://github.com/sonata-project/SonataMediaBundle/blob/master/composer.json#L25

**Comment by JHGitty** [comment]
related to https://github.com/sonata-project/SonataMediaBundle/pull/794

**Comment by jordisala1991** [comment]
Fixed on latest release of MediaBundle 3.4.0. ClassificationBundle no longed a required dependency
